items/bootstrap-3.3.5/css/style.css

```diff
@@ -227,18 +314,6 @@ label.star:before {
   font-family: FontAwesome;
 }
 
-/*Stars to display rating on the profile page. Need a better star image*/
```

Comment:
This set of css displays the star rating of the instructor on their profile page. If you remove it, the stars won't show up